_posts/2020-07-12-ksf-lead-after-first-weekend.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: news
+title: Kenilworth Sixth Form lead after the first weekend of matches
+---
+
+After the first 24 matches, team KSF from Kenilworth Sixth Form head up the
```

Comment:
Maybe worth noting here that it's the matches _of the SR2020 virtual competition_?

Reply:
639140e